locales/en.yml

```diff
@@ -30,4 +30,5 @@ search_results_counter: results found
 search_results_exclude_long_term: Hide long-running series
 search_results_exclude_permanent: Hide permanent sights
 search_resutls_remove_all_filters: Remove all filters
-search_results_searched_label: 'You looked up:'
\ No newline at end of file
+search_results_searched_label: 'You looked up:'
+warning_no_city_selected: Choose a valid municipality or region
```

Comment:
Nit: Consider adding a blank line at the end of each `yml` file in this directory to get cleaner diffs in the future. (Notice the "No newline at end of file" icon)